.github/workflows/continous-integration-workflow.yaml

```diff
@@ -21,7 +21,7 @@ jobs:
         run: sudo apt update
 
       - name: Install Java and Node
-        run: sudo apt-get install --yes default-jre default-jdk npm
+        run: sudo apt-get install --yes default-jre default-jdk nodejs
```

Comment:
Do we need to add nodejs repo first? [here](https://github.com/nodesource/distributions/blob/master/README.md#installation-instructions)
``` yaml
   - bash: "curl -sL https://deb.nodesource.com/setup_10.x | sudo -E bash -"
            displayName: Add nodejs repo
```

Reply:
The repo is installed as part of the script (we're raising a bug about this)